docs/specs/git.yml

```diff
@@ -27,4 +27,9 @@ outputs:
       "enableContribution": true,
       "editLink": "https://github.com/docascode/docfx-test-dependencies/blob/git-test/README.md"
     }
+  testdata1.json: |
```

Comment:
this breaks other builds

Reply:
open #2970 to fix it, because of login issue...